freeipa/src/main/resources/freeipa-salt/salt/freeipa/init.sls

```diff
@@ -18,6 +18,16 @@ net.ipv6.conf.lo.disable_ipv6:
       - {{ host['fqdn'] }}
 {% endfor %}
 
+{% if salt['pillar.get']('freeipa:hosts') > 1 %}
+/cdp/ipahealthagent/freeipa_cluster_node:
```

Comment:
I'm not sure it's a good idea to put a new folder under `/`. Couldn't we just put it under `/opt` somewhere?

Reply:
/cdp already existed from cluster proxy so seemed like a reasonable place to put it.
These already exist:
/cdp/bin/reverse-tunnel.sh
/cdp/bin/reverse-tunnel-values.sh
/cdp/bin/update-reverse-tunnel-values.sh